src/poll/awakener.rs

```diff
@@ -20,14 +19,13 @@ use crate::sys;
 /// woken up.
 ///
 /// Only a single `Awakener` should active per `Poller` instance, the `Awakener`
-/// can be cloned using [`try_clone`](Awakener::try_clone) if more are needed.
-/// What happens if multiple `Awakener`s are registered with the same `Poller`
-/// instance is undefined.
+/// can be cloned using [`try_clone`] if more are needed. What happens if
+/// multiple `Awakener`s are registered with the same `Poller` instance is
```

Comment:
Btw, I wonder, why is this behaviour undefined?

Reply:
Mainly to leave room for other possible implementations, it doesn't do anything wrong currently.

Reply:
Ah okay. Otherwise, this pr looks good!